modules/installation-configuration-parameters.adoc

```diff
@@ -938,6 +942,26 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`standard_LRS`, `premium_LRS`, or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`compute.platform.azure.osDisk.diskEncryptionSet.resourceGroup`
+|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites. This resource group should be different than the resource group where you install the cluster to avoid deleting your Azure encryption key when the cluster is destroyed. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
+|String, for example `production_encryption_resource_group`.
+
```

Comment:
compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId is an optional field that can set the subscription ID of a disk encryption set in another subscription. By default, the current subscription ID is used.

Reply:
Does this parameter also exist in `controlPlane.platform...` as well? or is it only for compute

Reply:
I have added this field to the configuration parameters table and the sample yaml file. PTAL